tests/unit/components/power-select/before-options-test.js

```diff
@@ -0,0 +1,28 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import Ember from 'ember';
+
+const { Controller, set } = Ember;
+
+moduleForComponent(
+  'power-select/before-options',
+  'Unit | Component | power select before options',
+  {
+    unit: true
+  }
+);
+
```

Comment:
What is a unit component test vs an integration? Not sure I've seen this distinction.